lua/nordic/tests/init.lua

```diff
@@ -0,0 +1,51 @@
+local U = require('nordic.utils')
+
+M = {}
+
+local function pretty_string(value)
+    if not U.is_table(value) then return tostring(value) end
+    local output = '{'
+    for k, v in pairs(value) do
+        output = output .. '' .. tostring(k) .. ': ' .. pretty_string(v) .. ', '
+    end
+    if #output == 1 then
+        return output .. '}'
+    else
+        return output:sub(1, -3) .. '}'
+    end
+end
+
+function M.assert_eq(left, right, message)
```

Comment:
Did you get this function from some other module?

Reply:
It's inspired by Rust's `assert_eq` macro, but all the code is new.

Reply:
Sweet.  Maybe we should rewrite the theme in Rust?...

Reply:
I have been tempted at times: https://github.com/KillTheMule/nvim-rs ;)